content/gophercon-2017/writing-a-successful-gophercon-proposal.md

```diff
@@ -0,0 +1,100 @@
+The reality of our online connected world is many talks are recorded and widely circulated.
+Our audience will not be happy to attend a conference composed of talks they think they've already seen.
+
+If you've presented a talk at a meetup or other conference and plan to propose it for GopherCon, by all means do so, but think about how you can incorporate the feedback you received and how the landscape may have changed since you gave your presentation last.
+
+# Writing a good GopherCon proposal
+
+The review team have no access to any other information about you, or your proposal save what you have written in the title, abstract, and details fields.
+Your job is to convince the reviewers that you can deliver an engaging, relevant, and informative presentation using only those three fields.
+
+## Title
+
+Keep it short, pithy, and to the point.
+Maybe it could be a little punny, or a little cryptic, but please avoid 8 shocking things that make your proposal read like a Buzzfeed headline.
+
+## Abstract
+
+Just the facts.
+One sentence that describes the topic that you'll be talking about, and one sentence that describes what the audience will take away from listening to your talk or partipating in your workshop, is all you need.
+But, just like a tweet, make those sentences count.
+
+## Detail
+
+The details section is where many proposals have traditionally come up short.
+To be blunt, many proposals across the years have not included enough detail for the reviewers to judge them according to the selection criteria.
+
+What we encourage speakers to do every year is including in their proposal:
+
+1. A paragraph or two of introduction, ideally the introduction you plan to use in your talk.
+2. An outline of the presentation itself. Bullet points are fine. The goal of including an outline is twofold, to demonstrate to the readers that you've thought about how your talk will flow from the questions you pose in your introduction, to the answers in your conclusion.
+3. A paragraph or two of conclustion, again ideally this would be the conclustion you plan to use in your talk.
+
+There is no required word limit, but if you haven't writen at least 400-500 words, you're probably selling your talk short and putting yourself at a disadvantage. 
+Simply put, proposals will less detail are rarely chosen over proposals with a lot of detail, we cannot stress this enough.
+
+# Addressing the selection criteria
+
+The review team score each proposal according to the following review criteria:
+
+- **Relevance**: Is this topic interesting to an audience of Go programmers? What is it about the topic that is unique to its use of Go?
+- **Novelty and Originality**: Does the talk provide the audience with new information? Preference will be given to presentations not previously available to a wide audience.
+- **Knowledge**: Has the speaker demonstrated a strong understanding of their subject?
```

Comment:
Yeah I'd also avoid the word "expert" otherwise it might invoke imposter syndrome, certainly does for me(I don't mean this in a bad way :), just giving the reason for not using that word).